spec/quizzy/user/repository_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "spec_helper"
+require "quizzy/user/repository"
+require "support/test_db"
+
+module Quizzy::User
+  describe Repository do
+    let(:repository) { TestDb.instance.repository(Repository) }
+
+    describe "#create" do
+      it "creates a user" do
+        TestDb.inside_transaction do
+          entity = Entity.new
+
+          entity.uid = "uid"
+          entity.provider = "google"
+          entity.name = "Foo Bar"
+          entity.email = "some@email.com"
+          entity.created_at = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.